Projekty/Projekt1/Jung_Kaczmarek_Marciniak/Milestone_2.ipynb

```diff
@@ -0,0 +1,2420 @@
+{
```

Comment:
<p>Można tak, ale ZDECYDOWANIE wygodniej byłoby użyć po prostu <code>sklearn.preprocessing.OneHotEncoder</code> (ponieważ te zmienne były binarne, to warto by to zrobić z parametrem <code>drop='first'</code> aby nie dodawać większej liczby kolumn niż potrzeba).</p><p> </p><p>Wygodnie by tutaj było również zrobić pipeline (to samo się dotyczy użycia OneHotEncoder w komórkach poniżej). </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/150/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>